python/taichi/lang/impl.py

```diff
@@ -367,13 +367,17 @@ def reset():
 
 @taichi_scope
 def static_print(*args, __p=print, **kwargs):
+    """The print function in Taichi scope. This function is called
```

Comment:
Note that docstring usually starts with a one line short-line description

Reply:
Sure. fixed.